langgraph/repo_analysis.py

```
@@ -0,0 +1,138 @@
+    
+    def get_full_readme(repo_path: Path) -> str:
+        """Extract the complete README content"""
+        readme_patterns = ['README.md', 'README.rst', 'README.txt', 'README']
+        for pattern in readme_patterns:
+            readme_path = repo_path / pattern
+            if readme_path.exists():
+                try:
+                    with open(readme_path, 'r', encoding='utf-8', errors='ignore') as f:
+                        return f.read().strip()
+                except Exception:
+                    continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)